common/src/main/java/com/microsoft/identity/common/internal/broker/PackageHelper.java

```diff
@@ -182,7 +182,11 @@ public static Signature[] getSignatures(final PackageInfo packageInfo) {
             if (packageInfo.signingInfo == null) {
                 return null;
             }
+            //Adding linke here: https://developer.android.com/reference/android/content/pm/SigningInfo#getSigningCertificateHistory()
+            //if the package has multiple signers the certificates may never be changed....
```

Comment:
What is the relevance of who is signing these certificates? This is the broker package, the certificate used to sign this package would be issued by the Microsoft's internal root CA? Which I am assuming eSTS trusts. Is my assumption incorrect?

Reply:
I think you followed up in Teams chat on this one.  The reason I linked to this... is that if the signature did change that would mean the app would no longer have access to any prior tokens from the cache.  Multiple signatures is uncommon but alos a potential complication.... when we chatted with Google... they basicallyi told us they don't test all of the AOSP features relative to multi-signing.... which seems to indicate that folks don't rely on it in practice.